src/api/spec/models/bs_request/find_for/group_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'rails_helper'
+
+RSpec.describe BsRequest::FindFor::Group do
+  describe '#all' do
+    let(:user) { create(:confirmed_user) }
+    let(:group) { create(:group) }
+    let(:target_package) { create(:package) }
+    let(:target_project) { target_package.project }
+    let(:source_package) { create(:package) }
+    let(:source_project) { source_package.project }
+
+    shared_examples 'has a request' do
+      let(:another_target_package) { create(:package) }
+      let(:another_target_project) { another_target_package.project }
+
+      it { expect(BsRequest::FindFor::Group.new(group: group.title).all).to include(request) }
```

Comment:
[Nitpick] could do something like this to avoid repeating the class constant all the time:
```ruby
let(:klass) { BsRequest::FindFor::Group }
it { expect(klass.new(group: group.title).all).to include(request) }
```